### PR TITLE
Fix/promql multi expr/srikanth

### PR DESCRIPTION
**Pull request by srikanthops**
* Added support for multiple expressions in a given graph
* Added workload for all exporters
* Added more exporters to akka
* Added JVM 
* Added AKKA HTTP

### CODE REVIEW

File: enterprise-suite/templates/es-grafana.yaml [hunk 1]
```diff
@@ -4,7 +4,141 @@ kind: ConfigMap
 metadata:
   name: exporter-graphs-cm
 data:
-{{ (.Files.Glob "es-grafana/*").AsConfig | indent 2 }}
```
Comment (dbrinegar) [inline enterprise-suite/templates/es-grafana.yaml]: please use this Glob and separate files, as before

File: enterprise-suite/templates/es-grafana.yaml [hunk 2]
```diff
@@ -4,7 +4,141 @@ kind: ConfigMap
...
+  # default setting
+  ernie.json: |
```
Comment (dbrinegar) [inline enterprise-suite/templates/es-grafana.yaml]: delete this file

File: enterprise-suite/templates/es-grafana.yaml [hunk 3]
```diff
@@ -4,7 +4,141 @@ kind: ConfigMap
...
+    []
+  akka-exporter.json: |
```
Comment (dbrinegar) [inline enterprise-suite/templates/es-grafana.yaml]: put these in separate files

File: docs/es/all.yaml [hunk 4]
```diff
@@ -82,71 +82,137 @@ data:
     []
   akka-exporter.json: |
     [
-      {"graphName":"Running Actors", "promQL":"akka_actor_running_actors{workload=\"$workloadId\"}"},
-      {"graphName":"Sent Messages", "promQL":"irate(akka_actor_sent_messages{workload=\"$workloadId\"}[1m])"},
-      {"graphName":"Processed Messages", "promQL":"irate(akka_actor_processed_messages{workload=\"$workloadId\"}[1m])"},
-      {"graphName":"Unhandled Messages", "promQL":"akka_actor_unhandled_message{workload=\"$workloadId\"}"},
-      {"graphName":"Actor Failures", "promQL":"akka_actor_actor_failure{workload=\"$workloadId\"}"},
-      {"graphName":"Dead Letters", "promQL":"akka_actor_dead_letter{workload=\"$workloadId\"}"}
+      {"graphName":"Running Actors", "promQL":["akka_actor_running_actors{workload=\"$workloadId\"}"]},
```
Comment (dbrinegar) [inline docs/es/all.yaml]: don't change all.yaml like this, this is built by the release script

**Comment by srikanthops** [comment]
looks like this is old way to do it.. I will send out new pull request.

**Pull request closed without merging**